Make.inc

```diff
@@ -42,6 +42,9 @@ USE_SYSTEM_PATCHELF:=0
 # Link to the LLVM shared library
 USE_LLVM_SHLIB := 1
 
+# Use the builtins from compiler-rt
+USE_COMPILER-RT :=0
```

Comment:
I'd be inclined to use underscores in variable names rather than dashes, for consistency's sake